### ISSUE TITLE
Weights drive link

### ISSUE THREAD

**Issue by ApurvaDatkhile** [KupynOrest/DeblurGAN#228]
The drive link provided to download the weights is not working

**Comment by Ttoooong** [comment]
yep,I am also unable to download the pre-training weights through the provided connection, who knows how to get down? Wait for a good Samaritan

**Comment by Ahmadreza1365** [comment]
You can download the images from the link below and train it yourself. 

https://drive.google.com/file/d/1CPMBmRj-jBDO2ax4CxkBs9iczIFrs8VA/view

Please see: 

https://github.com/KupynOrest/DeblurGAN/issues/222#issuecomment-1704701870
